.github/workflows/actions.yml

```diff
@@ -304,13 +304,181 @@ jobs:
+      INSTALL_PATH: ${{ github.workspace }}/build/Install
+      LIBS_DOWNLOAD_PATH: ${{ github.workspace }}/../3rd-party
+      VCVARS_SCRIPT_PATH: 'C:/Program Files (x86)/Microsoft Visual Studio/${{ matrix.msvc-year }}/Enterprise/VC/Auxiliary/Build/${{ matrix.vcvars-script }}'
+      ARTIFACT_FILE: 'SuperCollider-${{ needs.lint.outputs.sc-version }}-${{ matrix.artifact-suffix }}'
+      INSTALLER_FILE: 'SuperCollider-${{ needs.lint.outputs.sc-version }}-${{ matrix.installer-suffix }}'
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: prepare timestamp for cache
+        id: current-date
+        run: |
+          echo "::set-output name=stamp::$(date '+%Y-%m-%d')"
+          echo "::set-output name=week::$(date '+%V')"
+      - name: cache qt
+        id: cache-qt
+        uses: actions/cache@v1
+        with:
+          path: ../Qt
+          key: ${{ runner.os }}-v1-${{ matrix.os-version }}-${{ matrix.qt-version }}-qt${{ matrix.qt-arch }}
+      - name: install qt using aqtinstall
+        uses: jurplel/install-qt-action@v2
+        with:
+          modules: 'qtwebengine'
+          version: ${{ matrix.qt-version }}
+          arch: ${{ matrix.qt-arch }}
+          cached: ${{ steps.cache-qt.outputs.cache-hit }}
+      - name: install libsndfile
+        run: choco install libsndfile ${{ matrix.chocolatey-options }}
+      - name: download fftw
+        shell: bash
+        env:
+          FFTW_PATH: ${{ env.LIBS_DOWNLOAD_PATH }}/fftw
+        run: |
+          mkdir -p $FFTW_PATH && cd $FFTW_PATH
+          curl -L ${{ matrix.fftw-url }} -o fftw.zip
+          7z x fftw.zip -y
+          # add env var for subsequent steps
+          echo 'FFTW_PATH<<EOF' >> $GITHUB_ENV
+          echo $FFTW_PATH >> $GITHUB_ENV
+          echo 'EOF' >> $GITHUB_ENV
+      - name: create fftw msvc library
+        if: matrix.vcvars-script
+        shell: cmd
+        env:
+          FFTW_ARCH: ${{ matrix.fftw-arch }}
+        run: |
+          call "%VCVARS_SCRIPT_PATH%"
+          cd "%FFTW_PATH%"
+          lib.exe /machine:%FFTW_ARCH% /def:libfftw3f-3.def
+      - name: move fftw
+        shell: bash
+        run: |
+          mv $FFTW_PATH "C:/Program Files/fftw"
+      - name: install asio sdk
+        shell: bash
+        env:
+          ASIO_PATH: ${{ env.LIBS_DOWNLOAD_PATH }}/asio_sdk
+        run: |
+          mkdir -p $ASIO_PATH && cd $ASIO_PATH
+          curl -L http://www.steinberg.net/sdk_downloads/asiosdk2.3.zip -o asio.zip
+          7z x asio.zip -y
+          mv ASIOSDK2.3 $GITHUB_WORKSPACE/external_libraries/portaudio/asiosdk
+      - name: configure
+        shell: bash
+        run: |
+          export PATH=$Qt5_DIR/bin:$PATH
+
+          mkdir $BUILD_PATH && cd $BUILD_PATH
+          
+          BUILD_CONFIG=
+          if [[ ${{ startsWith(matrix.cmake-generator, 'MinGW') }} == true ]]; then BUILD_CONFIG="-D CMAKE_BUILD_TYPE=Release"; fi
```

Comment:
oh really? cmake would [produce a warning](https://github.com/supercollider/supercollider/runs/2002780046?check_suite_focus=true#step:11:87) for msvc... If you're convinced it was clearer, I'm happy to revert this change, just making sure.

Reply:
ah wow i haven't seen that warning before... personally, i like to specify the build type twice in build instructions, because many cmake users don't understand the difference between single and multi config generators, or forget that it makes a difference. in the case that someone switches their `-G` argument, this can save headaches. i've seen this in build instructions for other projects too, and i've seen experienced cmake users recommend it, so it's a little surprising to see cmake warning about it. i actually feel quite strongly that we should specify it twice to be honest.

Reply:
all right, sounds good!